code/ship/ship.cpp

```diff
@@ -6876,6 +6881,7 @@ void ship_subsys::clear()
 	system_info = NULL;
 
 	parent_objnum = -1;
+	parent_subsys_index = -1;
```

Comment:
Just confirming that this value doesn't need to be reset anywhere else?

Reply:
Right.  It is only accessed in places where it's guarded by the cache validity flag.